### PR TITLE
JENKINS: Dump coverity report to log.

### PR DESCRIPTION
**Pull request by yosefe**

### CODE REVIEW

**Comment by mellanox-github** [comment]
Test FAILed.
See http://bgate.mellanox.com/jenkins/job/gh-ucx-pr__NEW/126/ for details.

**Comment by mellanox-github** [comment]
Test PASSed.
See http://bgate.mellanox.com/jenkins/job/gh-ucx-pr/2720/ for details.

**Comment by mellanox-github** [comment]
Test PASSed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/2079/ for details (Mellanox internal link).

File: contrib/test_jenkins.sh [hunk 1]
```diff
@@ -218,9 +218,10 @@ if [ -n "$JENKINS_RUN_TESTS" ]; then
     rm -f jenkins_sidelinks.txt
     echo 1..1 > coverity.tap
     if [ $nerrors -gt 0 ]; then
+        cov-format-errors --dir $cov_build --emacs-style
         echo "not ok 1 Coverity Detected $nerrors failures # $cov_url" >> coverity.tap
     else
-        echo ok 1 Coverity found no issues >> coverity.tap
+        echo "ok 1 Coverity found no issues" >> coverity.tap
```
Comment (shamisp) [inline contrib/test_jenkins.sh]: I don't think it is really required ?

Comment (yosefe) [inline contrib/test_jenkins.sh]: not realy needed, it's just for consistency with previous line, i can remove it if it's very critical

Comment (shamisp) [inline contrib/test_jenkins.sh]: no need , I just was curious

**Pull request merged**